src/PostCampaign.php

```diff
@@ -456,34 +490,6 @@ public static function replace_font_aliases( array $css_vars ): array {
 		return $css_vars;
 	}
 
-	/**
```

Comment:
I couldn't find any usages of this, I assume it's not used anymore. @pablocubico can you confirm?

Reply:
No, I think the only use was in the line you deleted in this same PR.
If I recall correctly, this was here to satisfy a weird requirement, we had a spec for the hover colors (the "campaign_primary_color" was actually the hover color), but not for the idle state. 🤷